utils/vqm2blif/src/base/cleanup.h

```diff
@@ -19,6 +19,10 @@ extern int buffers_elim, inverts_elim, oneluts_elim;
 
 void netlist_cleanup (t_module* module);
 
+void remove_node ( t_node* node, t_node** nodes, int original_num_nodes );
+
+void reorganize_module_node_list(int original_number_of_nodes, int number_of_nodes_eliminated, t_node** module_node_list, t_module* module);
```

Comment:
Comment what these functions do here (in the header) so people calling them just have to read the header.  The comment should explain what each input to the function is, and what the function does (but doesn't have to explain *how* the function does it).

Reply:
Additional comments have been added now.